params/config.go

```diff
@@ -88,8 +89,45 @@ var (
 		},
 	}
 
-	TestChainConfig        = &ChainConfig{AvalancheContext{common.Hash{1}}, big.NewInt(1), DefaultFeeConfig, false, big.NewInt(0), big.NewInt(0), common.Hash{}, big.NewInt(0), big.NewInt(0), big.NewInt(0), big.NewInt(0), big.NewInt(0), big.NewInt(0), big.NewInt(0), NetworkUpgrades{big.NewInt(0)}, PrecompileUpgrade{}, UpgradeConfig{}}
-	TestPreSubnetEVMConfig = &ChainConfig{AvalancheContext{common.Hash{1}}, big.NewInt(1), DefaultFeeConfig, false, big.NewInt(0), big.NewInt(0), common.Hash{}, big.NewInt(0), big.NewInt(0), big.NewInt(0), big.NewInt(0), big.NewInt(0), big.NewInt(0), big.NewInt(0), NetworkUpgrades{}, PrecompileUpgrade{}, UpgradeConfig{}}
+	TestChainConfig = &ChainConfig{
+		AvalancheContext:    AvalancheContext{common.Hash{1}, snow.DefaultContextTest()},
```

Comment:
Can we update the formatting for this so that both values are on two different lines? Also, now that I think about it, the first value in `AvalancheContext` is the `blockchainID` which is included in `snow.Context` so we can actually remove the `BlockchainID` parameter entirely.

Reply:
yup nice catch, removed `BlockchainID`